cmd/test_load.go

```diff
@@ -39,6 +40,7 @@ type loadedTest struct {
 	preInitState   *lib.TestPreInitState
 	initRunner     lib.Runner // TODO: rename to something more appropriate
 	keyLogger      io.Closer
+	moduleResolver *modules.ModuleResolver
```

Comment:
Looking into the usage of this property in all cases, it sets the value using the value of the `runner,` which already exists as a `test.initRunner`. Am I missing something, or could we avoid introducing the duplication?

Reply:
`test.initRunner` is a lib.Runner interface https://github.com/grafana/k6/blob/65b2f27d1a0612846467e48418f469984724e007/lib/runner.go#L54 and it doesn't provide direct access to the ModuleResolver type that is a specific of the [js.Runner](https://github.com/grafana/k6/blob/65b2f27d1a0612846467e48418f469984724e007/js/runner.go#L47-L48) implementation.

It sounds like this is not clear enough from the code, do you think a comment somewhere could help?

Reply:
Ah, I see 👍

No I prefer to avoid commenting on this. It's just a lack of the PR context. In Code it should be fine.